chromadb/ingest/__init__.py

```diff
@@ -0,0 +1,33 @@
+from abc import ABC, abstractmethod
+import pulsar.schema as schema
+
+
+class AddEmbedding(schema.Record):
+    _avro_namespace_ = 'chromadb.ingest'
+    id = schema.Union([schema.String(), schema.UUID(), schema.Bytes(16)])
+    embedding = schema.Array(schema.Float())
+    metadata = schema.Map(schema.String(), schema.String())
+    object_reference = schema.String()
```

Comment:
for my understanding - what is this?

Reply:
Pulsar topic schema for a message:

- ID = embedding ID.
- Embedding = embedding vector
- Metadata = Metadata map
- Object Reference = pointer to embedded object (image, audio, document) in object storage.

But now that I say it, I realize that can be just another a metadata key. So I'll remove the object reference.